cmake/jsoncpp.cmake

```diff
@@ -26,7 +26,8 @@ else()
     # JSON_USE_INT64_DOUBLE_CONVERSION preprocessor macro. Doing so,
     # not only gets rid of the implicit conversion error that clang-10 produces
     # but also forces safer behavior in general.
-    set(JSONCPP_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DJSON_USE_INT64_DOUBLE_CONVERSION")
+    find_package(Threads REQUIRED)
+    set(JSONCPP_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DJSON_USE_INT64_DOUBLE_CONVERSION ${CMAKE_THREAD_LIBS_INIT}")
```

Comment:
This is only needed for the `ubuntu1604.clang.ossfuzz` compilation test. `ubuntu1604.clang.ossfuzz` is not able to build `jsoncpp` because of missing `pthread`. I agree that this is not very clean, I will try to find a better solution. Also `${CMAKE_THREAD_LIBS_INIT}` are linking flags and right now it is just put as `CXX_FLAGS`.

Reply:
Solved by upgrading to jsoncpp 1.9.3. In this version it is possible to disable the build of the examples. See https://github.com/ethereum/solidity/pull/9383